src/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.props

```diff
@@ -28,25 +28,26 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Publish everything under wwwroot, all JSON files, all config files and all Razor files -->
-    <Content Include="wwwroot\**" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
-    <Content Include="**\*.config" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
-    <Content Include="**\*.json" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
+    <Content Condition=" '$(DisableWebSdkContent)' == '' " Include="wwwroot\**" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+    <Content Condition=" '$(DisableWebSdkContent)' == '' " Include="**\*.config" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
+    <Content Condition=" '$(DisableWebSdkContent)' == '' " Include="**\*.json" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
 
     <!-- Set CopyToPublishDirectory to Never for items under AppDesignerFolder ("Properties", by default) to avoid publishing launchSettings.json -->
     <Content Update="$(AppDesignerFolder)\**" CopyToPublishDirectory="Never" Condition="'$(AppDesignerFolder)' != ''"/>
-    
+
     <!-- Remove Content items from other item types (in a way that CPS understands) -->
-    <None Remove="wwwroot\**;**\*.json;**\*.config" />
-    <Compile Remove="wwwroot\**" />
-    <EmbeddedResource Remove="wwwroot\**" />
-    
+    <None Condition=" '$(DisableWebSdkContent)' == '' " Remove="wwwroot\**" />
+    <None Remove="**\*.json;**\*.config" />
+    <Compile Condition=" '$(DisableWebSdkContent)' == '' " Remove="wwwroot\**" />
+    <EmbeddedResource Condition=" '$(DisableWebSdkContent)' == '' " Remove="wwwroot\**" />
+
     <!-- Keep track of the default content items for later to distinguish them from newly generated content items -->
-    <_ContentIncludedByDefault Include="@(Content)" />
+    <_ContentIncludedByDefault Condition=" '$(DisableWebSdkContent)' == '' " Include="@(Content)" />
 
   </ItemGroup>
 
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.Default.props"
-      Condition="Exists('$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.Default.props')"/>
+      Condition=" '$(DisableWebSdkContent)' == '' And Exists('$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.Default.props')"/>
```

Comment:
I thought we decided to keep the imports here. I am fine either way. If we are moving this, then we shouldn't add this to the globs file just to keep the globs logic separate.

Reply:
I kept the imports here.